fission-cli/library/Fission/CLI/Handler/User/Register.hs

```diff
@@ -81,10 +83,20 @@ createAccount ::
   , m `Raises` Username.Invalid
   , Show (OpenUnion (Errors m))
   )
-  => m Username
-createAccount = do
-  username <- ensureM $ mkUsername <$> Prompt.reaskNotEmpty' "Username: "
-  email    <- Email <$> Prompt.reaskNotEmpty' "Email: "
+  => Maybe Username
+  -> Maybe Email
+  -> m Username
+createAccount maybeUsername maybeEmail = do
+  username <- case maybeUsername of
+    Nothing ->
+      ensureM $ mkUsername <$> Prompt.reaskNotEmpty' "Username: "
+    Just uname ->
+      return uname
+  email    <- case maybeEmail of
+    Nothing ->
+      Email <$> Prompt.reaskNotEmpty' "Email: "
+    Just mail ->
+      return mail
```

Comment:
💅 Super super minor style thing.

I'd either give these more room to breathe...

```haskell
createAccount maybeUsername maybeEmail = do
   username <- case maybeUsername of
     Nothing    -> ensureM $ mkUsername <$> Prompt.reaskNotEmpty' "Username: "
     Just uname -> return uname

   email <- case maybeEmail of
     Nothing  -> Email <$> Prompt.reaskNotEmpty' "Email: "
     Just mail -> return mail
```

...or use a helper...

```haskell
createAccount maybeUsername maybeEmail = do
   username <- maybeUsername `otherwise` do
    ensureM $ mkUsername <$> Prompt.reaskNotEmpty' "Username: "

   email <- maybeEmail `otherwise` do
     Email <$> Prompt.reaskNotEmpty' "Email: "

  -- ...
  where
    otherwise :: Monad m => Maybe a -> m a -> m a 
    otherwise Nothing action = action
    otherwsise (Just val) _= return val
```